9.hello-docker-compose/components/messagebus.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: messagebus
```

Comment:
This is now called pubsub not messagebus in the samples

Reply:
Updated.